knowledge/__openerp__.py

```diff
@@ -0,0 +1,39 @@
+"""Knowledge Management System module."""
```

Comment:
this one probably, linked to this change: 

https://github.com/OCA/maintainer-quality-tools/commit/d8f265b93039a7f0199dcea3a3b92998db62def6